phys2bids/cli/run.py

```diff
@@ -125,6 +125,11 @@ def _get_parser():
                           type=str,
                           help='full path to store channels plot ',
                           default='')
+    optional.add_argument('-yml', '--participant-yml',
+                          dest='yml',
+                          type=str,
+                          help='file with info needed to generate participant.tsv file ',
+                          default='participants.yml')
```

Comment:
Do we need a default? in that case, shouldn't this default be  ``phys2bids/heuristics/participant.yml``?

Reply:
I wrote `participants.yml` because the tsv file is `participants.tsv`. I know the one I sent you was `participant.yml`.

Reply:
Actually i was saying that maybe we don't need a default, since the participants function has an option when the file does not exist
```suggestion
                          default='l')
```

Reply:
Done! Thanks for your suggestion!